docs/_includes/Tiered-Storage-on-Alluxio/pin-file.md

```diff
@@ -1,5 +1,5 @@
 ```java
-AlluxioFile file = AlluxioFileSystem.open("/myFile");
-SetStateOptions pinOpt = new SetStateOptions.Builder(ClientContext.getConf()).setPinned(true);
-AlluxioFileSystem.setState(file, pinOpt);
+AlluxioURI uri = new AlluxioURI("/myFile");
+SetAttributeOptions pinOpt = SetAttributeOptions.defaults().setPinned(true);
+FileSystem.Factory.get().setAttribute(uri, pinOpt);
```

Comment:
Could you make the example four lines and assign `FileSystem.Factory.get()` to a variable? We don't want to suggest users to always call that to access a `FileSystem` object.